modules/tinymce/src/core/main/ts/fmt/ApplyFormat.ts

```diff
@@ -122,6 +118,10 @@ const applyFormat = (ed: Editor, name: string, vars?: FormatVars, node?: Node |
     const newWrappers: Element[] = [];
     let contentEditable = true;
 
+    const isWrappingBlockFormat = FormatUtils.isBlockFormat(format) && format.wrapper;
+    const isNonWrappingBlockFormat = FormatUtils.isBlockFormat(format) && !format.wrapper;
+    const isWrapNameFormat = FormatUtils.isInlineFormat(format) || FormatUtils.isBlockFormat(format);
```

Comment:
This doesn't look correct? I block format is not something that wraps content. Should it be using is `isWrappingBlockFormat` above instead?

Reply:
Looking at the existing code again, I realised I don't actually need this variable so have just removed it